frontend/src/pages/auth/login/index.tsx

```diff
@@ -92,7 +92,7 @@ const LoginPage: React.FC = () => {
   };
 
   const openRegisterPage = () => {
-    // navigate({ to: '/register' });
+    navigate({ to: '/register' });
```

Comment:
We should also implement redirect uri
```suggestion
    // TODO: implement redirect uri logic
    navigate({ to: '/register' });
```